### ISSUE TITLE
msjnc doesn't start

### ISSUE THREAD

**Issue by datr** [madscientist/msjnc#10]
Just tried the script on a fresh build of Ubuntu 13.04. The script exits without an error code but nothing else seems to happen.

I'm afraid I don't know much perl but adding in some extra logging it seems to get to line 1657:

```
my $window = Gtk2::Window->new('toplevel');
```

And exits. Any ideas as to what might be happening?

**Comment by slintz** [comment]
Yup - same here: Upgraded from kubuntu 12.x to (gnome) ubuntu 14.x and MS-JNC just stopped working. Much hair pulling before I found this thread, which saved my bacon! Adding the [--fg] option on the command line works like a charm, and [deanproxy]'s advice to put it into the [ms-jnc.desktop] file makes it all sweetness and cream.

Now, I just wonder why this is a problem in the first place? I doubt it's something super particular to the Mad Scientist's work - but I don't have so much time to investigate. I have money to earn...
